cumulusci/core/flowrunner.py

```diff
@@ -790,7 +802,7 @@ def __call__(self, **options):
         if cache_key in self.cache.results:
             return self.cache.results[cache_key].return_values
 
-        task_config = self.cache.flow.project_config.tasks[self.task_name]
+        task_config = self.cache.project_config.tasks[self.task_name]
```

Comment:
@davidmreed we should also pass the step-specific project config below on line 812

Reply:
👍 Good catch.